ui/view/settings.py

```diff
@@ -26,8 +27,8 @@ class SettingsUI:
         "Belgian Daisy": BoardLayout.BELGIAN_DAISY,
     }
 
-    def __init__(self, on_close):
-        self.config = Config.from_default()
+    def __init__(self, config, on_close):
+        self.config = config if config else Config.from_default()
```

Comment:
nit: `config or Config.from_default()` is equivalent

Reply:
I changed game config to be passed in, so config settings represent the current settings instead of always being set from default when opening.